### ISSUE TITLE
!upload command?

### ISSUE THREAD

**Issue by CuddleBear92** [Just-Some-Bots/MusicBot#110]
right now the bot does download ever file pretty much right?

cant we have an !upload command that uploads the mp3 file into the chat? i think that would be nice!
either !upload [Link] to upload the song in question (needs to download if it doesnt have it) 
or just !upload to upload the one playing ATM. 

since there is an upload size limit i also think the bot should link to where it got the file if the file is too large for Discord.

i mean the bot is just a normal user pretty much right? so it could do anything a normal user does right? isnt this possible?

**Comment by XCanG** [comment]
You think right, but I also now trying to add local playing, because bot hosted on my home PC and I have some local music.
I think for me this task now impossible, just because need to find comment in Discord and download file (currently this bot does not have native code from download, it use **youtube-dl** to download and it's another story).

**Comment by imayhaveborkedit** [comment]
Discord limits file uploads to 8MB.  For most 2-3 minute songs this is fine, but once you get up around 5 minutes it starts being an issue.  There's also the issue of certain formats being uncompressed, meaning large filesizes for shorter songs.  The thing with youtube links is that the direct link is gigantic.  About 950 characters long.  That's half the discord message length limit.  The link also expires after some time (the one I tested says it expires in 6 hours from when I generated it), and includes the ip of the machine the bot is running on.  

Basically, there's no good way to do this, and I'm not sure I want to do this anyways.  There are tons of youtube downloading plugins for every browser.  If you really want to download a youtube video of potentially questionable quality, that's your job.
